site/source/tutorials/006_deploying_to_production/index.md

```diff
@@ -0,0 +1,143 @@
+
+{% section %}
+
+## Taking full control
+
+{% task 'Take full control of the build process.' %}
+
+The build tool is designed to cover the most common use cases for developing and deploying Dojo 2 applications. There are times, however, when you need to take full control of the deployment process. To achieve this level of control, a project can be *ejected* from the `dojo` tool and all of its configuration information exported.
+
+**Note:** This is a *non-reversable* process. Once a project has been ejected, the `dojo` tool can no longer be used to manage a project.
+
+To eject a project, use the `dojo eject` command. You will be prompted to ensure that you understand that this is a non-reversible action. Entering **'y'** will begin to export process. The export process generates a new directory - `config` that contains all of the exported configuration information for each of the `dojo-cli` tools that the project had been using. The process will also install some additional dependencies that the project now requires.
+
+The project is now configured to be managed as a webpack project. Changes can be made to the build configuration by altering the `config/build-webpack/webpack.config.js` file. A build can then be triggered by running `webpack`'s build command and providing the configuration file. That command is: `./node_modules/.bin/webapack --config ./config/build-webpack/webpack.config.js`.
+
+{% section %}
+
+## Summary
+
+For many software projects, preparing an application for deployment to production often involves creating complicated configuration files and at least a little bit of experimentation. Dojo 2, on the other hand, provides a single command, `dojo build` that is used both during development as well as for production deployments.
+
+While the `dojo build` command addresses many use cases, a few configuration options are necessary to support certain development and deployment scenarios. The `--locale`, `--supportedLocales`, and `--messageBundles` arguments are used to configuration the applications internationalization support. The `--debug` argument allows the results of the build to be examined in more depth. Finally, the `--disableLazyWidgetDetection` argument can be used to optimize how an application loads widgets.
+
+To ensure repeatable builds, all of the configuration options for the `dojo` CLI commands can be stored in the `.dojorc` file. This JSON file is the ideal choice to use when it is necessary to ensure that consecutive builds are executed with the same parameters.
+
+In the event that the development team requires a higher level of control than Dojo 2's build system offers, a project can be exported via the `dojo eject` command. This non-reversible command writes all of the `dojo` CLI commmands' configuration information to the file system, providing a solid starting point.
```

Comment:
command writes all of the `dojo` CLI commmands' configuration information to the file system, providing a solid starting point.  ->  command exports all of the `dojo` CLI commands' configuration information, providing a solid starting point.